package.json

```diff
@@ -23,7 +23,7 @@
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
     "start": "electron background.js",
-    "setupmac": "node ./scripts/setupmac.js",
+    "setup": "node ./scripts/setup.js",
```

Comment:
The reason the mac setup script was kept separate was because Windows will be run entirely differently. No hypervisor will be used because you can't on Windows 10 Home. Instead, we'll need to use Windows Subsystem for Linux (WSL) to make it work. That means no minikube.

Wouldn't the same apply for Linux? Why not run k3s right on the local linux system? I'm open to discussion on this.

Reply:
Interesting! I didnt know Windows Home version didnt have hyper-X available. To be fair I had even forgotten that the Home version existed, its so crippled and the Pro version was so cheap that I thought people just ignored it!